### PR TITLE
Updated readme with latest commands

### LINKED ISSUE #26: Update README.md
Readme needs updated documentation for all newly introduced commands

### PR DESCRIPTION
**Pull request by ashutoshjv661**
Added latest commands 
To fix #26

### CODE REVIEW

**Review by makoteq — CHANGES_REQUESTED** [review]

**Comment by makoteq** [comment]
Hey, PR is looking good just change mentioned thing

File: README.md [hunk 1]
```diff
@@ -18,6 +20,14 @@ npx metaxa start <path> <delay>
 ```
 _where < path > is a path to your .txt file with text to read and < delay > is the speed of the words in words per minute (WPM)_
 
+2. Show number of words, sentences, and absolute path of file in < file_path >
+``` bash
+metaxa stats <file_path>
+```
+3. Show text from text wrapped with '' or "" in < text >, with delay between words in < delay > (default 250WPM)
+``` bash
```
[the remark this answers is not in the document]
Comment (ashutoshjv661) [reply README.md:28]: sure , can you help me here.
read is a bash command thats why its getting blue colour. 
Should I remove bash formatting and put red for "<" and ">"

**Comment by ashutoshjv661** [comment]
Hi @makoteq as "read" was a bash command it was highlighted in blue. I have used cmd for the fenced code.

**Comment by makoteq** [comment]
image
other thing - plese add [] brackets to < delay > just as you did for `read` command
@ashutoshjv661

**Review by makoteq — APPROVED** [review]

**Comment by makoteq** [comment]
after that we are good to go

**Comment by ashutoshjv661** [comment]
sure I will do it right away

**Comment by ashutoshjv661** [comment]
I have added the changes ^_^

**Comment by makoteq** [comment]
Great job!

**Comment by ashutoshjv661** [comment]
Thanks for the merge ^_^

**Pull request merged**